packages/paste-website/src/pages/form-elements/select/index.mdx

```diff
@@ -0,0 +1,942 @@
+
+To support internationalization, avoid starting a sentence with the label and using the field to finish it since sentence structures vary across languages. For example, use “Call log expiration date” or “How long should logs be stored?”. Don’t use “Remove logs after:”.
+
+Give users enough information in help text to prevent input and formatting errors. Keep it concise and scoped to information that will help with validation. For example, use help text if a password field has specific requirements that a user should know prior to filling it out.
+
+If you need to use more than 2 sentences to explain a field, link out to supporting docs or trigger a popover instead.
+
+### Required or optional field identifier
+
+Ask for information only when needed. Consider removing the field otherwise.
+
+In a form where there are fewer optional fields than required, use an optional indicator to show users which fields they don’t have to fill out. Otherwise, use required indicators to show users which fields they must fill out.
+
+If a field is required and there’s no safe default selected value, default to a disabled option that tells users to choose an option (_e.g._, “Choose categories”). However, if the field is optional and there’s no safe default, keep the option enabled so that users can unselect.
+
+### Required field key
+
+Use the [required field key]() component if you use the `required` prop in your form fields. This component should appear at the top of your form to explain what the required indicator means on each form field. By default, use the words ‘Indicates a required field’ in the key.
+
+### Validation
+
+Disable or remove any options in a select that would result in an error if possible. Validate form fields on form submission.
+
+Validating a form field when the user leaves the current field (on blur) can be helpful to check for required field entry. However, this can be frustrating to users who tab through controls to navigate a page, and to screen reader users, who might not be able to detect that an error occurred on blur.
+
+Don’t prevent form submission by disabling the submit button. An error message can give more information than a disabled button can to help users figure out which fields are invalid.
+
+### Errors
+
+Disable or remove any options in a select that would result in an error if possible. If an error occurs, use inline error text to explain how to fix an error.
+
+Ideally, help text should have enough information to help users prevent errors. If help text exists and you need to show an error, the error text should replace and repeat the help text until the error has been resolved.
+
+Error text should:
+
+<UnorderedList>
+  <ListItem>
+    Be actionable. Explain how to fix an error and if reasonable, why it happened so that it might also be prevented in
+    the future.
+  </ListItem>
+  <ListItem>
+    Be concise and simple, maybe even more than normal. Avoid jargon. Try to keep error text to 2 sentences or fewer.
```

Comment:
~~, maybe even more than normal~~

Reply:
😅 Def C+P the copy from this and didn't really read through it. Consider it removed. ✅